src/appConfig.js

```diff
@@ -80,7 +80,8 @@ configuration.queryparams =
         'GageStatsServicesBounds': '/stations/Bounds?xmin={0}&xmax={1}&ymin={2}&ymax={3}&geojson=true',
         'FlowAnywhereEstimates': '/models/FLA/estimate?state={0}',
         'FlowAnywhereGages': '/arcgis/rest/services/IowaStreamEst/FlowAnywhere/MapServer/1/query?geometry={0},{1}&geometryType=esriGeometryPoint&inSR=4326&spatialRel=esriSpatialRelIntersects&outFields=regions_local.Region_Agg,reference_gages.site_id,reference_gages.site_name,reference_gages.da_gis_mi2,reference_gages.da_pub_mi2,reference_gages.lat_dd_nad,reference_gages.long_dd_na&returnGeometry=false&returnIdsOnly=false&returnCountOnly=false&returnZ=false&returnM=false&returnDistinctValues=false&f=pjson',
-        'NSSRegions': '/regions/'
+        'NSSRegions': '/regions/',
```

Comment:
```suggestion
        'regions': '/regions/',
```

Let's rename this to be more consistent with the names of other queryparams.

Reply:
@amedenblik, from looking at the streamstatsservices query params farther up starting with SS, I believe that NSSRegions is consistent as is. Would you still like this to be changed?

Reply:
Yeah let's go ahead and change it. There are numerous queryparams farther up that use NSS (examples include "RegressionScenarios", "statisticsGroupLookup", "statisticsGroupParameterLookup", "estimateFlows") but we don't preface them with "NSS".